test/functional/wallet_reindex.py

```
@@ -0,0 +1,108 @@
+
+import time
+
+from test_framework.blocktools import COINBASE_MATURITY
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+)
+BLOCK_TIME = 60 * 10
+
+class WalletReindexTest(BitcoinTestFramework):
+    def add_options(self, parser):
```

Ruff (0.12.2)

Missing return type annotation for public function `add_options`

Add return type annotation: `None`

(ANN201)